src/getAppSyncConfig.js

```diff
@@ -197,8 +197,25 @@ export default function getAppSyncConfig(context, appSyncConfig) {
 
       const defaultTemplatePrefix = getDefaultTemplatePrefix(template);
 
-      const requestTemplate = request || `${defaultTemplatePrefix}.request.vtl`;
-      const responseTemplate = response || `${defaultTemplatePrefix}.response.vtl`;
+      const requestTemplate = !isNil(request)
+              ? request
+              : !isNil(cfg.defaultMappingTemplates.request)
+                ? cfg.defaultMappingTemplates.request
+                : `${defaultTemplatePrefix}.request.vtl`;
+
+      if (requestTemplate === false) {
+        throw new Error('Sorry, this plugin does not support Direct Lambda resolvers yet.');
+      }
+
+      const responseTemplate = !isNil(response)
+              ? response
+              : !isNil(cfg.defaultMappingTemplates.response)
```

Comment:
@abdala make sure to use the `?` since `defaultMappingTemplates` could be undefined. 

`!isNil(cfg.defaultMappingTemplates?.response)`

Also, merge both `if`s and keep one Error, please

Then I think we are good to merge.

Thank you again